app/views/sequences/_activity_list.html.haml

```diff
@@ -8,7 +8,7 @@
           = truncate(activity.name_with_id, :length => 20, :omission => "…")
           %ul.menu
             %li.edit= link_to "Edit", edit_activity_path(activity) if can? :update, activity
-            %li.run= link_to "Run", activity_path(activity)
+            %li.run= link_to "Preview", sequence_path(@sequence) + activity_path(activity)
```

Comment:
@scytacki There isn't a `preview` method for sequences, so I just joined the sequence path to the activity path to form a URL for running the activity within the sequence. Is that good enough do you think? Or should we add an actual preview for sequences?